app/models/user.rb

```diff
@@ -2,4 +2,8 @@ class User < ApplicationRecord
   has_many :posts
   has_many :comments
   has_many :likes
```

Comment:
- [ ] According to the given [ERD diagram](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/images/blog_app_erd.png), the `posts, comments, and likes` should have a `foreign_key: 'author_id'`. Please kindly add it 🙃 😉

Reply:
Well resolved ✔️